### ISSUE TITLE
DOMNodeInserted event is not triggered

### ISSUE THREAD

**Issue by guogaigai** [nwjs/nw.js#4778]
I encounter a issue when i use nw.js0.13.0 , my code is below :   frame.addEventLIstener("DOMNodeInserted",beforeLoad);  beforeLoad is a callback . 
when i use nw.js0.12.3 this callback is executed . but after i update nw version to nw.js0.13.0 , this event is not executed. Does someone know the reason ? thank you

**Comment by marcgfx** [comment]
just came looking for a solution to this exact problem. I'm using 0.14.2

**Comment by rogerwang** [comment]
Does it work in the corresponding Chromium version?

**Comment by vsemozhetbyt** [comment]
May be they drop these deprecated events finally? See:

https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Mutation_events
https://developers.google.com/web/updates/2012/02/Detect-DOM-changes-with-Mutation-Observers

**Comment by marcgfx** [comment]
works in chrome/firefox/safari. I was thinking there might be a setting in nw.js to enable these events?

**Comment by vsemozhetbyt** [comment]
Well, it is strange.

1. I've tested this code in the Google Chrome 52.0.2718.0 canary (64-bit):

```javascript
document.body.addEventListener('DOMNodeInserted', () => {console.log('Node added.')});
document.body.appendChild(document.createElement('div'));
```
After the code execution I've also added another node via copy/paste in the console. The result:

![ch](https://cloud.githubusercontent.com/assets/10393198/14863677/5ee424f8-0cbf-11e6-8fc5-5bce6bf55748.png)

In Google Chrome nodes are inserted and event listener is executed.

2. I've done the same in the NW.js 0.15.0-beta1. The result:

![nw](https://cloud.githubusercontent.com/assets/10393198/14863711/8cceaf5a-0cbf-11e6-8ac4-835ba77e26d2.png)

In NW.js nodes are inserted, but event listener is not executed.
